packages/console/src/utilities/validator.ts

```diff
@@ -14,3 +14,17 @@ export const uriValidator = (verifyBlank = true) => {
     return true;
   };
 };
+
+export const uriOriginValidator = ({ allowBlank = true }) => {
+  return (value: string) => {
+    if (!allowBlank && value.trim().length === 0) {
+      return true;
```

Comment:
seems that if `allowBlank` is false, we should return false here?

Reply:
this is a confusing naming, should rename back to `verifyBlank`